### PR TITLE
labels for enterprise-tenant

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/enterprise-tenant/managing-user-hierarchies.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ If the delegated user also needs to manage specific devices, the admin user must
 Delegation works only inside user management and does not have any implication to other places.
 {{< /c8y-admon-info >}}
 
-### Troubleshooting sub-users
+### Troubleshooting sub-users {#troubleshooting-subusers}
```
Comment (BeateRixen) [inline content/enterprise-tenant/managing-user-hierarchies.md:83]: > sub-users

File: content/enterprise-tenant/usage-and-billing.md [hunk 2]
```diff
@@ -254,17 +254,17 @@ Note that an audit record is created for every change of the number of instances
...
 {{< product-c8y-iot >}} platform servers by default work at UTC timezone. Other time zones are also supported by the platform and can be selected by the service provider at installation time. Thus, the general metering functionality is also guaranteed for non-UTC server time zones.
 {{< /c8y-admon-important >}}
 
 The tenant usage statistics are collected on a daily base according to the beginning of day (`BOD`) and the end of day (`EOD`), which are defined by the server timezone. As a result, if the local time zone of a user is different from the server timezone, an operation triggered by the user may be assigned to a different day according to the server time.
 
-#### Examples
+#### Examples {#examples}
 
-##### Request counting - Example 1
+##### Request counting - Example 1 {#request-counting--example-1}
```
Comment (BeateRixen) [inline content/enterprise-tenant/usage-and-billing.md:267]: not sure about this one; the hyphen is missing but 2 hyphens for the spaces have been added

File: content/enterprise-tenant/usage-and-billing.md [hunk 3]
```diff
@@ -276,7 +276,7 @@ The tenant usage statistics are collected on a daily base according to the begin
 The request will be billed to the day 25.08.2020 as this is the server time of the request handing.
 <br><br>
 
-##### Request counting - Example 2
+##### Request counting - Example 2 {#request-counting--example-2}
```
Comment (BeateRixen) [inline content/enterprise-tenant/usage-and-billing.md:279]: see above

File: content/enterprise-tenant/usage-and-billing.md [hunk 4]
```diff
@@ -288,7 +288,7 @@ The request will be billed to the day 25.08.2020 as this is the server time of t
 The request will be billed to the day 26.08.2020 as the server time is the same as the device time.
 <br><br>
 
-##### Microservice resource billing - Example 1
+##### Microservice resource billing - Example 1 {#microservice-resource-billing--example-1}
```
Comment (BeateRixen) [inline content/enterprise-tenant/usage-and-billing.md:291]: see above

File: content/enterprise-tenant/usage-and-billing.md [hunk 5]
```diff
@@ -301,7 +301,7 @@ The request will be billed to the day 26.08.2020 as the server time is the same
 The resources will be assigned mainly to the day 26.08.2020 as according to the UTC time zone the microservice was active for 14 hours that day and for 10 hours the next day. This might be a bit different from what a user expects as from his perspective the microservice was active for 12 hours each day.
 <br><br>
 
-##### Microservice resource billing - Example 2
+##### Microservice resource billing - Example 2 {#microservice-resource-billing--example-2}
```
Comment (BeateRixen) [inline content/enterprise-tenant/usage-and-billing.md:304]: see above

File: content/enterprise-tenant/usage-and-billing.md [hunk 6]
```diff
@@ -314,7 +314,7 @@ The resources will be assigned mainly to the day 26.08.2020 as according to the
 From the user perspective the microservice was subscribed for 8 hours at 26.08.2020 but at server time it was 2 hours before EOD of 25.08.2020 and 6 hours after BOD at 26.08.2020.
 <br><br>
 
-##### Microservice resource billing - Example 3
+##### Microservice resource billing - Example 3 {#microservice-resource-billing--example-3}
```
Comment (BeateRixen) [inline content/enterprise-tenant/usage-and-billing.md:317]: see above

File: content/enterprise-tenant/managing-user-hierarchies.md (hunk 1 above)
Comment (rssag) [inline content/enterprise-tenant/managing-user-hierarchies.md:83]:
```suggestion
### Troubleshooting sub-users {#troubleshooting-sub-users}
```

File: content/enterprise-tenant/usage-and-billing.md (hunk 2 above)
Comment (rssag) [inline content/enterprise-tenant/usage-and-billing.md:267]:
```suggestion
##### Request counting - Example 1 {#request-counting---example-1}
```

File: content/enterprise-tenant/usage-and-billing.md (hunk 3 above)
Comment (rssag) [inline content/enterprise-tenant/usage-and-billing.md:279]:
```suggestion
##### Request counting - Example 2 {#request-counting---example-2}
```

File: content/enterprise-tenant/usage-and-billing.md (hunk 4 above)
Comment (rssag) [inline content/enterprise-tenant/usage-and-billing.md:291]:
```suggestion
##### Microservice resource billing - Example 1 {#microservice-resource-billing---example-1}
```

File: content/enterprise-tenant/usage-and-billing.md (hunk 5 above)
Comment (rssag) [inline content/enterprise-tenant/usage-and-billing.md:304]:
```suggestion
##### Microservice resource billing - Example 2 {#microservice-resource-billing---example-2}
```

File: content/enterprise-tenant/usage-and-billing.md (hunk 6 above)
Comment (rssag) [inline content/enterprise-tenant/usage-and-billing.md:317]:
```suggestion
##### Microservice resource billing - Example 3 {#microservice-resource-billing---example-3}
```

File: content/enterprise-tenant/managing-user-hierarchies.md (hunk 1 above)
Comment (rssag) [reply content/enterprise-tenant/managing-user-hierarchies.md:83]: added

File: content/enterprise-tenant/usage-and-billing.md (hunk 2 above)
Comment (rssag) [reply content/enterprise-tenant/usage-and-billing.md:267]: should be three. added

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**